index.html

```diff
@@ -527,6 +527,37 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcrtpencodingparameters-video-keyframe">
+    <h3>
+      RTCRtpEncodingParameters extensions for requesting the generation of a key frame.
+    </h3>
+    <p>
+      The {{RTCRtpEncodingParameters}} dictionary is defined in
+      [[WEBRTC]]. This document extends that dictionary with
+      additional members to request generation of a key frame by the encoder.
+    </p>
+    <pre class="idl">partial dictionary RTCRtpEncodingParameters {
+  boolean requestKeyframe = false;
```

Comment:
wondering if a default of "false" is appropriate here. This is more similar to maxBitrate which is unset unless explicitly configured?